### PR TITLE
Add feature tests

### PR DESCRIPTION
**Pull request by richardhallett**
This is for implementing https://github.com/crosscite/doi-metadata-search/issues/106
It uses headless chrome to perform the tests.
The only consideration im not sure of is, the example dois to use for when doing tests.

### CODE REVIEW

File: views/works/item.haml [hunk 1]
```diff
@@ -30,22 +30,22 @@
     - puts metrics
     .panel-footer
       - unless views < 1
-        %span
+        %span.metrics-views
```
Comment (kjgarza) [inline views/works/item.haml]: This was part of the old implementation. Back then the icons and metrics would disappear completely if there were no metrics. However, the new mockups https://datacite.atlassian.net/wiki/spaces/TEC/pages/531529771/Display+stats+in+DataCite+Search  the icons stay present and the values change to "--". And @daslerr to clarify the "--" value should appear only when there is no data, but if a resource has 20 views but 0 downloads, it should say 0 not "--", right?

File: spec/features/item_data_page_spec.rb [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
-  # end
+  describe 'metrics tests' do
+
+    it 'when metrics should exist has metrics' do
+      visit '/works/10.7272/q6g15xs4'
+      expect(page).to have_css 'span.metrics-views'
+      expect(page).to have_css 'span.metrics-downloads'
+    end
+
+    it 'when metrics should not exist doesnt have has metrics' do
```
Comment (kjgarza) [inline spec/features/item_data_page_spec.rb]: The test for the chart should be very similar. I'm putting the charts within a divs with 

-           .row#downloads-chart
-           .row#view-chart

File: views/works/item.haml (hunk 1 above)
Comment (richardhallett) [reply views/works/item.haml]: We can do a check for something more specific, or alternativly we add a class when there is values, I kinda prefer that tbh as you could arguably then apply a style specifically to if there is no values.
So maybe .metrics-views-empty .metrics-views-notempty or something.
Food for thought, either way the actual testing shouldn't make any difference.

**Comment by richardhallett** [comment]
This has been updated to reflect the new generated layout.
There is no need to actually test for the difference between datasets that have no data and datasets that do, so we just check for the existance of the display.
In addition because of the tabs we check for just the existance of charts in general.

**Pull request merged**